code/game/objects/machinery/doors/railing.dm

```diff
@@ -7,6 +7,8 @@
 	opacity = FALSE
 	open_layer = CATWALK_LAYER
 	closed_layer = WINDOW_LAYER
+	smoothing_behavior = null
+	smoothing_groups = null
 
```

Comment:
Shouldn't this be 

smoothing_behavior = NO_SMOOTHING
smoothing_groups = NONE

instead of null

Reply:
yep, should use the define.